index.js

```diff
@@ -95,6 +96,7 @@ class AbstractBtpPlugin extends EventEmitter {
         ws.once('message', async (binaryAuthMessage) => {
           try {
             authPacket = BtpPacket.deserialize(binaryAuthMessage)
+            debug('got auth packet', JSON.stringify(authPacket))
```

Comment:
Could be:

``` js
debug('got auth packet. packet=%j', authPacket)
```

That means that `JSON.stringify` (a very expensive operation) is only called when `DEBUG` is actually enabled.